src/third-party/discord-rpc/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required (VERSION 3.1.0)
+cmake_minimum_required (VERSION 3.5)
```

Comment:
Please don't modify third-party code. Those modifications will all be reverted when I update this library.